client/src/components/MainNavBar.vue

```diff
@@ -131,6 +137,11 @@ export default {
       window.location.href = this.$router.resolve({ name: 'register' }).href;
     },
 
+    navigateToPanel(event) {
+/*       event.preventDefault()
+      window.location.href = this.$router.resolve({ name: 'panel' }).href; */
+    },
```

Comment:
By panel, do you mean where we can see the services with their actions/reactions ?

Reply:
Yes that's it